lisa/platform_.py

```diff
@@ -22,6 +25,23 @@
 
 _get_init_logger = partial(get_logger, "init", "platform")
 
+PlatformStatus = Enum(
```

Comment:
It's not status, it's operations. For our requirement, the messages should be post operation, like initialized, prepared, deployed, deleted. If we need before operation, we can add it later.

Reply:
I was earlier planning to implement post operations but sometimes there is no specific time when operation is completed. For example, `_delete_environment` function with `wait_delete = false` has no specific time when the resource group will be completely deleted. Any ideas how to handle this case if we implement post operations?

Reply:
delete is special, but it's ok to dispatch a deleted message, even `wait_delete == false`. It means the environment is deleted from this run, although it may not be deleted physically.